ui/models/taskcluster.js

```diff
@@ -122,6 +122,7 @@ export default class TaskclusterModel {
           if (knownKinds.includes(action.kind) &&
             !actions.some(({ name }) => name === action.name) &&
             ((!action.context.length && !originalTask) ||
+              action.name === 'cancel-all' ||
```

Comment:
`action.context = []` for the cancel-all action, so this should not be necessary (it's not an exception to any rules..)

Reply:
I see. In that case, I'd have to change `tasksInContext` to return true when tagSetList is emtpy...

```jsx
static taskInContext(tagSetList, taskTags) {
  return tagSetList.some(tagSet => Object.keys(tagSet)
    .every(tag => taskTags[tag] && taskTags[tag] === tagSet[tag]),
  );
}
```

Reply:
I don't think so -- line 124 takes care of that.  The source of the error is that you're running "cancelAll" with a taskId, when "all" does not apply to any specific task.  So once `originalTask` is undefined, line 124 will be true and `taskInContext` won't be called.